lib/gym/options.rb

```diff
@@ -49,6 +49,11 @@ def self.plain_options
                                      env_name: "GYM_OUTPUT_DIRECTORY",
                                      description: "The directory in which the ipa file should be stored in",
                                      default_value: "."),
+        FastlaneCore::ConfigItem.new(key: :archive_path,
+                                     short_option: "-b",
+                                     env_name: "GYM_ARCHIVE_PATH",
+                                     description: "The directory in which the archive file should be stored in",
+                                     default_value: false),
```

Comment:
This shouldn't be `default_value: false`, but should be `optional: true` instead